docs/operator-guide/cd/inject-variables.md

```diff
@@ -0,0 +1,31 @@
+# Inject Variables Into Deploy Pipelines
+
+KubeRocketCI allows you to inject variables into deploy pipelines via the KubeRocketCI portal. With this feature, you can specify key-value pairs that will be injected into the deploy pipeline as environment variables. This enhancement allows you to configure application deployment settings directly within the KubeRocketCI portal. This page provides comprehensive guidelines on how to set key-value variables for your deploy pipelines.
```

Comment:
lets use parameters instead of variables, we need to provide business goal here instead of technical side